book/src/csi-driver-object.md

```diff
@@ -61,7 +61,7 @@ These are the important fields:
   - For more information see [Pod Info on Mount](pod-info.md).
 - `fsGroupPolicy`
   - This field was added in Kubernetes 1.19 and cannot be set when using an older Kubernetes release.
-  - This field is alpha.
+  - This field is beta.
```

Comment:
beta in 1.20

Reply:
I've added it here, too; however, should we do this with the other fields (in a separate PR)?

Reply:
Yes I think updating the other fields makes sense